overlays/tooling/default.nix

```diff
@@ -18,7 +18,11 @@ nixlib.lib.composeManyExtensions [
 
     ssh-for-each = final.writeShellScriptBin "ssh-for-each" ''
       for host in ${self}/hosts/*; do
-          ${final.openssh}/bin/ssh "''${host#'${self}/hosts/'}" ''${@}
+          host="''${host#'${self}/hosts/'}"
+          if [ "$host" != "groups" ];
```

Comment:
Why not just move things out of the directory? Why is it necessary to have this config live in directory named "hosts" and not something else? I really do not want to add exclusions here...

Reply:
Because the "groups" information is related to hosts that are getting deployed. I'm happy to instead get a directory structure like
```
/
| - flake.nix
...
\ - inventory
  \ - hosts
    | - attest.profian.com
    | - store.profian.com
    | - ...
  \ - groups
    \ - meta
      | - common.nix
    \ - steward.rdu.infra.profian.com
      | - default.nix
```

Or something like that?
Basically, I want to indicate that hosts info and groups info belong together, and hosts depend on groups, but this information should not be used by external repositories.